js/groups.js

```diff
@@ -269,22 +269,25 @@ GroupList = {
 		GroupDeleteHandler = new DeleteHandler('/apps/user_management/groups', 'groupname',
 			GroupList.hide, GroupList.remove);
 
-		//configure undo
-		OC.Notification.hide();
 		var msg = escapeHTML(t('user_management', 'deleted {groupName}', {groupName: '%oid'})) + '<span class="undo">' +
 			escapeHTML(t('user_management', 'undo')) + '</span>';
 		GroupDeleteHandler.setNotification(OC.Notification, 'deletegroup', msg,
-			GroupList.show);
+			GroupList.remove);
 
 		//when to mark user for delete
 		$userGroupList.on('click', '.delete', function () {
 			// Call function for handling delete/undo
-			GroupDeleteHandler.mark(encodeURIComponent(GroupList.getElementGID(this)).toString());
-		});
-
-		//delete a marked user when leaving the page
-		$(window).on('beforeunload', function () {
-			GroupDeleteHandler.deleteEntry();
+			var groupName = encodeURIComponent(GroupList.getElementGID(this)).toString();
+			OC.dialogs.confirm(
+				t('user_management', 'You are about to delete a group. This action can\'t be undone and is permanent. Are you sure that you want to permanently delete {groupName}?', {groupName: decodeURIComponent(groupName)}),
```

Comment:
can you add a space before the question mark in a subsequent PR ? @sharidas

Reply:
@settermjd can you advise here please?
I have never heard of putting a space before a question mark.

Reply:
never mind... I just checked the l10n files and apparently there is never a space.

so I've been doing this wrong all these years ?
in some languages you need a space there

Reply:
never mind... I just checked the l10n files and apparently there is never a space.

so I've been doing this wrong all these years ?
in some languages you need a space there

this one confirms it: https://english.stackexchange.com/a/4646

apparently the fact that I'm french has biased me: http://grammaire.reverso.net/5_1_10_les_espaces_et_la_ponctuation.shtml (see example sentences)

seems German also doesn't have the space: https://www.duden.de/sprachwissen/rechtschreibregeln/fragezeichen

Reply:
Ah, English, the inconsistent beast that it is.